web-app/js/portal/filter/BaseFilterPanel.js

```diff
@@ -84,6 +84,14 @@ Portal.filter.BaseFilterPanel = Ext.extend(Ext.Panel, {
     },
 
     _setExistingFilters: function() {
+    },
+
+    needsFilterRange: function() {
```

Comment:
I'm a bit unsure about this... I would've thought that the range/set of possible values is a property of the `Filter` itself?  Obviously this has implications for the UI, but can it be derived from the Filter rather than coded here?